packages/accordion/src/AccordionItem.ts

```diff
@@ -23,6 +24,13 @@ import chevronIconStyles from '@spectrum-web-components/icon/src/spectrum-icon-c
 
 import styles from './accordion-item.css.js';
 
+const chevronClass = {
```

Comment:
This is what's stumping me about whether or not we'd want to use a helper extension. If we don't need `SizedMixin` here, but we have sized _icons_, will that cause a problem?